.travis.yml

```diff
@@ -37,7 +37,9 @@ jobs:
         - mkdir -p "$HOME/.php-cs-fixer"
```

Comment:
@keradus 

We can probably also remove this then, and the corresponding caching directive?